target/snitch_cluster/README.md

```diff
@@ -1,24 +1,60 @@
-# Snitch cluster target
+# SNAX Cluster Target
 
+## Details
 The Snitch cluster target (`target/snitch_cluster`) is a simple RTL testbench
-around a Snitch cluster. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
+around a SNAX cluster. It is still the same as the Snitch cluster but with SNAX modifications. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
 
 The configuration parameters are documented using JSON schema. Documentation for the schema and available configuration options can be found in `docs/schema-doc/snitch_cluster/`).
 
 The cluster testbench simulates an infinite memory. The RISC-V ELF file to be simulated is
 preloaded using RISC-V's Front-End Server (`fesvr`).
 
-## Tutorial
+## Quick Start
```

Comment:
I don't like this section. Where do users have to look to make stuff? It explains the same as the detailed tutorial below, but it's not upfront about what is exactly being built.
This needs some introduction at the top, like:
```If you quickly want to get started with a HWPE mac engine, follow this section...```
Furthermore it gives us extra places to adapt documentation later.
If we change the mechanism later we have to adapt it in more than one place, which is a hassle.

Reply:
I was thinking that the Quick Start is literally quick easy steps to get something running. Like a summary at the top, before the detailed stuff. I can add an extra line to emphasize what it is for.

Reply:
Hmm I think it's not much of a hassle to change too much. It's still better to have this "quick start" summary imo. So I hope we can keep it first hehe.

Also I added more introduction stuff to this.